docs/enableexperimental.md

```diff
@@ -0,0 +1,36 @@
+# How to enable experimental features
+
+Fleet supports experimental features that are disabled by default and that can be enabled by the user.
+
+Enabling/disabling experimental feaures is done usign extra environment variales that are available when deploying rancher/fleet.
+
+## Enabling an experimental feature
+
+At the moment we're writing this document fleet has OCI storage as an experimental feature.
+### Enabling when installing fleet stand-alone
```

Comment:
nit: an extra line break here makes the general layout of this document look nicer to me, but this comment can be safely dismissed :)
```suggestion

### Enabling when installing fleet stand-alone
```